plexapi/base.py

```diff
@@ -926,6 +922,35 @@ def stop(self, reason=''):
         return self._server.query(key, params=params)
 
 
+class PlexHistory(object):
+    """ This is a general place to store functions specific to media that is a Plex history item.
+
+        Attributes:
+            accountID (int): The associated :class:`~plexapi.server.SystemAccount` ID.
+            deviceID (int): The associated :class:`~plexapi.server.SystemDevice` ID.
+            historyKey (str): API URL (/status/sessions/history/<historyID>).
+            viewedAt (datetime): Datetime item was last watched.
+    """
+
+    def _loadData(self, data):
+        self.accountID = utils.cast(int, data.attrib.get('accountID'))
+        self.deviceID = utils.cast(int, data.attrib.get('deviceID'))
+        self.historyKey = data.attrib.get('historyKey')
+        self.viewedAt = utils.toDatetime(data.attrib.get('viewedAt'))
+
+    def _reload(self, **kwargs):
+        """ Reload the data for the history entry. """
+        raise NotImplementedError('History objects cannot be reloaded. Use source() to get the source media item.')
+
+    def source(self):
```

Comment:
this is an unfortunate method name, causing conflict:
- https://github.com/pkkid/python-plexapi/pull/1335#issuecomment-1881698233

what inspired this method name? maybe rename to `origin` or `item` or `original`?

Reply:
@JonnyWong16: do you have answer? may give some ideas what other name to use here instead:
- https://github.com/pkkid/python-plexapi/pull/1335#issuecomment-1883539258